### PR TITLE
eng(multi-actions): honor model choice when possible

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

Instead of hard-coding the model used for multi-actions scheduling, we can use the assistant's model if it supports multi actions. Otherwise, we default to claude 3 opus.

### CODE REVIEW

File: front/lib/api/assistant/agent.ts [hunk 1]
```diff
@@ -422,6 +428,7 @@ export async function getNextAction(
   config.MODEL.function_call = forcedActionName ?? "auto";
   config.MODEL.provider_id = model.providerId;
   config.MODEL.model_id = model.modelId;
+  config.MODEL.temperature = 0.2;
```
Comment (spolu) [inline front/lib/api/assistant/agent.ts:431]: Meh

**Review by spolu — COMMENTED** [review]
Otherwise LGTM

File: front/lib/api/assistant/agent.ts (hunk 1 above)
Comment (spolu) [reply front/lib/api/assistant/agent.ts:431]: Should be the assistant config for now no?

Comment (spolu) [reply front/lib/api/assistant/agent.ts:431]: So we can play with it easily. Maybe in the future we’ll add a config

Comment (fontanierh) [reply front/lib/api/assistant/agent.ts:431]: > Should be the assistant config for now no?

Mmh fair. Although one could also argue we might want a different setting for scheduling vs generation. But for now let's use assistant's config indeed. Making a PR 👍

**Pull request merged**